src/components/PatronBlock/PatronBlockForm.js

```diff
@@ -14,8 +14,9 @@ import {
   TextArea,
   Checkbox,
   Datepicker,
-  AppIcon
 } from '@folio/stripes/components';
+
+import { AppIcon } from '@folio/stripes-core/src/components';
```

Comment:
That `AppIcon` deprecation warning in stripes-components is _super_ noisy, but I'd rather handle it in a separate PR after we get folio-org/stripes-core/pull/591 merged, assuming it looks correct to you. Then we can import `AppIcon` from `@folio/stripes/core` along with `TitleManager` below. Same for the import in `src/Users.js`. Sound good?